.github/workflows/ci.yml

```diff
@@ -42,6 +42,9 @@ jobs:
             php-version: '8.0'
           - phpunit-version: 8
             php-version: '8.1'
+          - phpunit-version: 8
+            php-version: '8.2'
+            composer-flags: '--ignore-platform-req=PHP'
```

Comment:
pls don't
if we claim support for a combination of given PHP / PHPUnit versions, but only with this flag - someone in their project will not be using that flag and face issue that CI is exposing here, without this flag.

some extra details in https://github.com/php-coveralls/php-coveralls/pull/346#issuecomment-1230079070

Reply:
ok, it's different. we wait for phpspec/ to release new version compact with 8.2. agree to proceed